bears/python/PyFlakesBear.py

```diff
@@ -6,8 +6,8 @@
 @linter(executable='pyflakes',
         use_stderr=True,
         output_format='regex',
-        output_regex=r'(?P<filename>.*):(?P<line>\d+):'
-                     r'[(?P<column>\d+):|?]*(?P<severity>)\s(?P<message>.*)\n',
```

Comment:
this line doesnt need to be modified.
changing it makes me waste time looking for a obscure change to the regex, when there is none.